src/components/filters/filter-panels/filter-panel-with-field-select.js

```diff
@@ -60,7 +61,7 @@ function FieldPanelWithFieldSelectFactory(
       ]);
 
       const fieldValue = useMemo(
-        () => (Array.isArray(filter.name) ? filter.name[0] : filter.name),
+        () => ((Array.isArray(filter.name) ? filter.name[0] : filter.name)),
```

Comment:
Still unclear why `prettier` thinks this is a good idea, might be worth formatting again to make sure it's the right format for this repo

Reply:
hmm not sure when is this added. I removed it, and lint didn't complain